### PR TITLE
Dot files and directories will be ignored

### PR DESCRIPTION
**Pull request by kyteinsky**
Mock files and directories starting with dot(.) will be ignored.
Test case added to route that is contained in .proxy.hcl file and should ideally return a 404 status.

### CODE REVIEW

**Comment by kyteinsky** [comment]
Solves #53

File: mocko-proxy/test/proxy.js [hunk 1]
```diff
@@ -25,4 +25,9 @@ module.exports = (mocko, _describe, it) => () => {
         const { status } = await mocko.post('/validate/query?foo=bar');
         expect(status).to.equal(201);
     });
+
+    it('should ignore files and directories starting with dot(.)', async () => {
+        const { status } = await mocko.post('/dot-route');
+        expect(status).to.equal(404);
```
Comment (gabriel-pinheiro) [inline mocko-proxy/test/proxy.js:31]: As the request returns a 404 code, the mocko.post will reject. You might need something like:
```js
let code;

await mocko.get('/dot-route')
    .then(({ status }) => code = status)
    .catch(({ response }) => code = response.status);

expect(code).to.equal(404);
```

**Review by gabriel-pinheiro — CHANGES_REQUESTED** [review]

File: mocko-proxy/test/proxy.js (hunk 1 above)
Comment (kyteinsky) [reply mocko-proxy/test/proxy.js:31]: I didn't go into the documentation for mocko.post(), thanks for pointing that out.
I will change it right away.

**Review by gabriel-pinheiro — APPROVED** [review]

**Comment by gabriel-pinheiro** [comment]
Thank you so much for you contribution, @kyteinsky ! 😄

**Comment by kyteinsky** [comment]
I learnt something today as well. I didn't know await and then-catch block could be used together. I always used try-catch with async-await.
Thank you for that.

**Pull request merged**